Makefile

```diff
@@ -0,0 +1,11 @@
+
+.PHONY: build lint
+
+build:
```

Comment:
missing clean beforehand

Reply:
I don't think it's needed, but maybe we can add another command that does both?